exercises/numbers/isEven/isEven.js

```diff
@@ -1,39 +1,23 @@
-/**
- * Returns true if the given number is even false otherwise.
- *
- * An integer is even if the remainder is 0 after dividing
- * by 2. Equivalently, an integer is even if it can be written
- * as 2*k for some integer k.
- *
- * If an integer is not even we call it "odd".
- *
- * Even: -10, -12, -104, 0, 16, 18, 2
- * Odd: -11, -13, -103, -1, 1, 17, 19, 3
- *
- * @param {number} num The input number
- * @returns {boolean} True if num is even and false otherwise
- */
+
+let num = {}
+
 function isEven(num) {
-  // Hint: See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Remainder
-  return _____;
+  if (num % 2 === 0)
```

Comment:
First, always use `{ ... }` with `if / else / for` statements. You _will_ make a mistake at some point.

Second, because `num % 2` evaluates directly to `true` or `false` you don't even need the conditional.

Consider:

```js
function isEven(num) {
  return n % 2 === 0;
}
```